system/mp/mpcomm.cpp

```diff
@@ -687,8 +698,9 @@ protected: friend class CMPPacketReader;
         // also in sendmutex
 
         ISocket *newsock=NULL;
-        unsigned retrycount = 10;
+        unsigned retrycount = 20;
```

Comment:
more loops of shorter duration to check for when the other (accept) thread connects to reduce the overall wait time.